resources/graphics.py

```diff
@@ -36,6 +36,9 @@
 				"pointerPivot",
 				"pointerCut",
 				"pointerNotEditable",
+                "pointerReplacePaths",
+                "pointerAddPaths",
+                "pointerRemovePaths",
```

Comment:
I think this feature would benefit greatly from some more specific icons :

- A "pointer with sets" icon for the initial drag from the SetEditor, using the icon we use to represent sets in there (or something like it).
- The +=- versions of the above, with a slightly brightened version of the sets icon.

Reply:
I've tried a couple of different icon treatments, a smaller set icon in the pointer in https://github.com/GafferHQ/gaffer/pull/5786/commits/a43f45d1b33882bca64fb96f0e8c61cd1b0c300b, and a larger set icon in https://github.com/GafferHQ/gaffer/pull/5786/commits/b5f3d6dc9f170f031f31f422d2f8b7aa41eb2e75.

I favor the larger icon for clarity and matching up roughly in size with the objects drag pointers. But the smaller one is a bit less jarring when starting the drag from the Set Editor.

Reply:
Definitely feels better with the new icons, and I prefer the larger icon too. I think something is off with either the line widths or the pixel alignment compared to the PathFilterUI though - the `=+-` signs don't match, and the border for the set icon feels thicker than for the objects.

Reply:
That was kind of strange, they were copy and pasted right from the other pointers. Anyways, I cleaned those line weights up and double checked the alignments in https://github.com/GafferHQ/gaffer/pull/5786/commits/d1451e49d21ea7c10f757e1f27d62ceed46f05af.